packages/synapse-react-client/src/components/widgets/query-filter/EnumFacetFilter/EnumFacetFilter.tsx

```diff
@@ -137,8 +145,13 @@ function _EnumFacetFilter(props: EnumFacetFilterProps) {
     )
     const valueNotSetFacetArray = partitions[0]
     const restOfFacetValuesArray = partitions[1]
+    const sortedValues = sortBy(restOfFacetValuesArray, fv =>
+      fv.displayText.toLowerCase(),
+    )
```

Comment:
May want to check if the facet column is a number/integer, and if so, parse before sorting (don't forget a special case for null). Otherwise we'll end up with issues like these (though unlikely to happen for most real date data) 

```javascript
_.sortBy(['2001', '2010', '2003', '20030', '2011', '20110'], x => x.toLowerCase())
//   -> [ '2001', '2003', '20030', '2010', '2011', '20110' ]
```

Reply:
parsing number for simple (not list) number type columns